src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -1938,6 +1967,248 @@
+  "com.minecolonies.research.effects.healthaddition.description": "Citizen HP +%d",
+  "com.minecolonies.research.effects.healingsaturationlimitaddition.description": "Healing Saturation Min %d",
+  "com.minecolonies.research.effects.workingdayhaddition.description": "Citizen Work Day +%dH",
+  "com.minecolonies.research.effects.blockplacespeedmultiplier.description": "Citizen Block Place Speed +%3$d%%",
+  "com.minecolonies.research.effects.levelingmultiplier.description": "Citizen XP Growth +%3$d%%",
+  "com.minecolonies.research.effects.sleeplessmultiplier.description": "Guards Need %3$d%% Less Sleep",
+  "com.minecolonies.research.effects.walkingmultiplier.description": "Citizen Walk Speed +%3$d%%",
+  "com.minecolonies.research.effects.archerarmormultiplier.description": "Archer Armor +%3$d%%",
+  "com.minecolonies.research.effects.moreoresmultiplier.description": "Miners Find +%3$d%% More Ores",
+  "com.minecolonies.research.effects.teachingmultiplier.description": "XP Gain When Studying +%3$d%%",
+  "com.minecolonies.research.effects.regenerationmultiplier.description": "Citizen Regeneration +%3$d%%",
+  "com.minecolonies.research.effects.growthmultiplier.description": "Child Growth Rate +%3$d%%",
+  "com.minecolonies.research.effects.armordurabilitymultiplier.description": "Guard Armor +%3$d%% Durability",
+  "com.minecolonies.research.effects.farmingmultiplier.description": "Farmers Harvest +%3$d%% Crops",
+  "com.minecolonies.research.effects.minimumstockmultiplier.description": "Buildings Can Minimum Stock %3$d%% More",
+  "com.minecolonies.research.effects.blockattacksmultiplier.description": "Knight Shield Blocking Chance +%3$d%%",
+  "com.minecolonies.research.effects.recipesmultiplier.description": "Workers Can Learn +%3$d%% More Recipes",
+  "com.minecolonies.research.effects.happinessmultiplier.description": "Citizen Happiness +%3$d%%",
+  "com.minecolonies.research.effects.tooldurabilitymultiplier.description": "Citizen Tools +%3$d%% Durability",
+  "com.minecolonies.research.effects.meleearmormultiplier.description": "Knights Armor +%3$d%%",
+  "com.minecolonies.research.effects.saturationmultiplier.description": "Citizen Saturation Per Meal +%3$d%%",
+  "com.minecolonies.research.effects.fleeingdamagemultiplier.description": "Guards Take -%3$d%% Damage When Fleeing",
+  "com.minecolonies.research.effects.blockbreakspeedmultiplier.description": "Citizen Block Break Speed +%3$d%%",
+  "com.minecolonies.research.effects.doublearrowsmultiplier.description": "Archer Multishot +%3$d%%",
+  "com.minecolonies.research.effects.piercingarrowsunlock.description": "Archers Gain Piercing II",
+  "com.minecolonies.research.effects.consumearrowsunlock.description": "Archers Use Arrows For +2 Damage",
+  "com.minecolonies.research.effects.shieldusageunlock.description": "Knights Unlock Shield Use",
+  "com.minecolonies.research.effects.plant2unlock.description": "Plantations Grow Two Plants at Once",
```

```suggestion
  "com.minecolonies.research.effects.plant2unlock.description": "Plantations Grow Two Crops at Once",
```
I don't really like the repetition.